### ISSUE TITLE
Use app-navigation-entry-utils-menu-button

### ISSUE THREAD

**Issue by stefan-niedermann** [owncloud/calendar#875]
the current solution looks bloaty and ugly and it does not scale very good. what about using app-navigation-entry-utils-menu-button like in news app?

**current**
![current](https://cloud.githubusercontent.com/assets/4741199/8801610/3f6ed86a-2fbc-11e5-9e71-ac0cceb88f5c.png)

**news-app**
![expected](https://cloud.githubusercontent.com/assets/4741199/8801616/4623bb08-2fbc-11e5-971d-40be0e15be12.png)

cc @owncloud/designers @jancborchardt

**Comment by MorrisJobke** [comment]
Oh yeah :) Would be really nice :)

**Comment by jancborchardt** [comment]
Yes, this is the plan for [Calendar+](https://github.com/libasys/calendarplus) as talked about with @libasys :)

**Comment by libasys** [comment]
I agree!

**Comment by libasys** [comment]
@jancborchardt please review a first try on http:://demo81.libasyscloud.de. I moved the complete settings dialog to the left sidebar! You can close new, edit calendar mode with ESC!

**Comment by jancborchardt** [comment]
@libasys can you open a work-in-progress pull request in your calendarplus repo? Then we can discuss and review there. :)

**Comment by jancborchardt** [comment]
@libasys but man, in general that looks much nicer already! :) Settings moved to the proper locations. Just two things I noticed:
* the dot-dot-dot menu should ideally be to the right of the sharing icon instead of on the left as it’s now
* the animation for the bottom left settings menu is missing

There are some other details like clickability of the dot-dot-dot-menu (I’ll fix that) and the loading overlay often overlaying the whole screen when it shouldn’t – but those we can fix separately. :)

**Comment by libasys** [comment]
@jancborchardt the more menu is now on the right side, the animation (Slide) is also integrated, i can't use the default animation of the settings menu, cause the left nav is loaded per ajax and so the cors js function not works! I am doing some commits on the rework branch, cause i am changed a lot of code on the server side and so it is not 100% save, that all functions works as expected! working with new libs parallel to old libs for migration, so be cool and wait, before submitting some PRs! Thanks for the feedback!
